WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/simplepayments/SimplePaymentsFragment.kt

```diff
@@ -70,9 +95,29 @@ class SimplePaymentsFragment : BaseFragment(R.layout.fragment_simple_payments) {
                 val df = DecimalFormat("#.##")
                 binding.textTaxLabel.text = getString(R.string.simple_payments_tax_with_percent, df.format(taxPercent))
             }
+            new.customerNote.takeIfNotEqualTo(old?.customerNote) { customerNote ->
+                binding.textCustomerNoteContent.text = customerNote
+                binding.textCustomerNoteContent.isVisible = customerNote.isNotEmpty()
+                binding.textEditCustomerNote.isVisible = customerNote.isNotEmpty()
+                binding.textAddCustomerNote.isVisible = customerNote.isEmpty()
+            }
+        }
+    }
+
+    private fun setupResultHandlers() {
+        handleResult<String>(SimplePaymentsCustomerNoteFragment.SIMPLE_PAYMENTS_CUSTOMER_NOTE_RESULT) {
+            viewModel.onCustomerNoteChanged(it)
         }
+    }
 
-        // TODO nbradbury - customer note
+    private fun showCustomerNoteEditor() {
+        val bundle = Bundle().also {
+            it.putString("customerNote", viewModel.viewState.customerNote)
+        }
+        findNavController().navigate(
+            R.id.action_simplePaymentsFragment_to_simplePaymentsCustomerNoteFragment,
+            bundle
+        )
     }
```

Comment:
As we use `safeargs` from Navigation Library, WDYT about using it over manually bundle creation? It'd be awesome too to use `navigateSafely`, just in case.

```suggestion
    private fun showCustomerNoteEditor() {
        SimplePaymentsFragmentDirections
            .actionSimplePaymentsFragmentToSimplePaymentsCustomerNoteFragment(viewModel.viewState.customerNote)
            .let { findNavController().navigateSafely(it) }
    }
```